### PR TITLE
Update dependencies, fix security vulnerabilities

### PR DESCRIPTION
**Pull request by swiftone**

### CODE REVIEW

**Comment by mraible** [comment]
@swiftone Can you please try adding `http://localhost:8080` as a login redirect URI to the app on https://samples-javascript.okta.com and restarting the job in Travis?

**Comment by swiftone** [comment]
Added and restarted

**Comment by mraible** [comment]
Sahweet! Tests pass!! 🎉

File: README.md [hunk 1]
```diff
@@ -4,10 +4,21 @@ This repository contains several sample applications that demonstrate various Ok
...
-- **Client Id** - The client ID of the SPA application that you created earlier. This can be found on the "General" tab of an application, or the list of applications.  This identifies the application that tokens will be minted for.
-- **Issuer** - This is the URL of the authorization server that will perform authentication.  All Developer Accounts have a "default" authorization server.  The issuer is a combination of your Org URL (found in the upper right of the console home page) and `/oauth2/default`. For example, `https://dev-1234.oktapreview.com/oauth2/default`.
+> **Tip**: You can also create an account using the [Okta CLI](https://github.com/oktadeveloper/okta-cli) and `okta register`. To create an app, run `okta apps create` and use the settings below.
+
+Register your application by selecting **Applications** > **Add Application**. On the next screen, choose **Single Page App** and click **Next**.
+
+On the following screen, edit the application settings. For Angular applications running in developer mode, the port number should be 4200. Configure your app as follows:
+
+* **Base URI**: `http://localhost:4200`
+* **Login redirect URI**: `http://localhost:4200/callback`
```
Comment (swiftone) [inline README.md:16]: I don't really recommend '/callback'.  This is not the default setting and assumes their app would have only one callback.

As deceptive as it is, I'd continue to use `/implicit/callback` until we change the default redirect_uri.  (at which point I'm advocating for `/login/callback`, though others have suggested `/oidc/callback`)

Is `4200` an angular convention?

Comment (mraible) [reply README.md:16]: We've been using `/callback` in all our 2020 blog posts. @aaronpk recommended it. Why does using it prevent you from having `/secondcallback`? 

Yes, 4200 is an Angular convention. React uses 3000 and Vue uses 8080.

Comment (swiftone) [reply README.md:16]: I like the concept of meaningful urls.  If I have a callback, and /callback, I'll assume they are the same, but if I have 2 callbacks, I have no idea which one is /callback.

Regardless, it doesn't match our defaults, so I'm questioning if we want to require the extra step and easy point of confusion.

Comment (swiftone) [reply README.md:16]: Another point is that we've had problems with devs accidentally blocking access to routes, if we know they have `/login` accessible without authentication then `/login/callback` is likely free of complication.

File: README.md [hunk 2]
```diff
@@ -4,10 +4,21 @@ This repository contains several sample applications that demonstrate various Ok
...
+
+On the following screen, edit the application settings. For Angular applications running in developer mode, the port number should be 4200. Configure your app as follows:
+
+* **Base URI**: `http://localhost:4200`
+* **Login redirect URI**: `http://localhost:4200/callback` 
+* **Logout redirect URI**: `http://localhost:4200` 
+
+Once you have completed the form, you will be given a **client ID**. You will also need the **issuer** value for your Okta org. 
+
+The **issuer** is the URL of the authorization server that will perform authentication.  All Developer Accounts have a "default" authorization server.  The issuer is a combination of your Org URL (found in the upper right of the console home page) and `/oauth2/default`. For example, `https://dev-133337.okta.com/oauth2/default`.
```
Comment (swiftone) [inline README.md:21]: If we don't have similar language in our other samples we should steal this.

**Comment by mraible** [comment]
@swiftone I updated [my branch](https://github.com/mraible/samples-js-angular/tree/update-dependencies) to use `/login/callback` and upgrade to Angular 10.1. Should I create a new PR or are you able to merge those changes into this one?

**Comment by swiftone** [comment]
@mraible - Assuming I git'ted correctly, your changes should now be in here, let me know if anything is missing.

**Comment by mraible** [comment]
@swiftone I noticed that the current custom login example has the following:

```ts
const oktaConfig = Object.assign({
  onAuthRequired: ({oktaAuth, router}) => {
    // Redirect the user to your custom login page
    router.navigate(['/login']);
  }
}, sampleConfig.oidc);
```

However, the [Angular SDK docs](https://www.npmjs.com/package/@okta/okta-angular#using-a-custom-login-page) seem to indicate it should be:

```ts
const oktaConfig = Object.assign({
  onAuthRequired: (oktaAuth, injector) => {
    const router = injector.get(Router);
    // Redirect the user to your custom login page
    router.navigate(['/login']);
  }
}, sampleConfig.oidc);
```

Which one is correct?

**Comment by aarongranick-okta** [comment]
@mraible the second one. The first argument is the Okta auth service, the second argument is the Angular injector (this changed in okta-angular 2.0 I think)

**Comment by mraible** [comment]
@aarongranick-okta In that case, can you please include [this commit](https://github.com/mraible/samples-js-angular/commit/8d4f83d0d92ea782569263023098100acfed2729) in this PR.

**Pull request merged**